### ISSUE TITLE
Feedback: How about showing Call LTP and Put LTP as well

### ISSUE THREAD

**Issue by dinu1389** [VarunS2002/Python-NSE-Option-Chain-Analyzer#22]

**Comment by VarunS2002** [comment]
@dinu1389 Added to feature requests

**Comment by VarunS2002** [comment]
@dinu1389 In the option menu you can enable "Dump entire Option Chain data to a `.csv` file".
LTP is shown in the `.csv` file.
Do you want it on the UI screen?

**Comment by dinu1389** [comment]
Yes @VarunS2002 it would be better cause we usually look at LTP in our trading platforms.

**Comment by dinu1389** [comment]
Thanks for your response :)

**Comment by VarunS2002** [comment]
> Yes @VarunS2002 it would be better cause we usually look at LTP in our trading platforms.

Okay I might add it some time. Until then you can refer the csv file or the website if you really need it :)
